CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 
 - Pretty print panic information.
 
+- Firecracker terminates with exit code 148 when a non-whitelisted syscall
```

Comment:
Add a deprecated header to CHANGELOG.md and specify that syscalls metrics will be removed in a future release. Also, add a new issue for removing the syscalls metric.

Reply:
Created an issue, but didn't add anything to the changelog, because I'm not sure yet about the approach we're going to take.